src/components/stepper/stepper.e2e.ts

```diff
@@ -223,35 +238,40 @@ describe("calcite-stepper", () => {
     expect(step2).not.toHaveAttribute("active");
     expect(step3).not.toHaveAttribute("active");
     expect(step4).not.toHaveAttribute("active");
+    expect(activeContent.textContent).toEqualHtml(step1content.textContent);
     await element.callMethod("endStep");
     await page.waitForChanges();
     expect(step1).not.toHaveAttribute("active");
     expect(step2).not.toHaveAttribute("active");
     expect(step3).not.toHaveAttribute("active");
     expect(step4).toHaveAttribute("active");
+    expect(activeContent.textContent).toEqualHtml(step4content.textContent);
   });
 
   it("navigates disabled items correctly with startStep and endStep methods", async () => {
     const page = await newE2EPage();
     await page.setContent(html`<calcite-stepper>
       <calcite-stepper-item item-title="Step 1" id="step-1" disabled>
-        <div>Step 1 content</div>
+        <div id="step-1-content">Step 1 content</div>
       </calcite-stepper-item>
       <calcite-stepper-item item-title="Step 2" id="step-2">
-        <div>Step 2 content</div>
+        <div id="step-2-content">Step 2 content</div>
       </calcite-stepper-item>
       <calcite-stepper-item item-title="Step 3" id="step-3">
-        <div>Step 3 content</div>
+        <div id="step-3-content">Step 3 content</div>
       </calcite-stepper-item>
       <calcite-stepper-item item-title="Step 4" id="step-4" disabled>
-        <div>Step 4 content</div>
+        <div id="step-4-content">Step 4 content</div>
```

Comment:
Looks like `step-1-content` and `step-4-content` aren't used. For clarity, I'd suggest removing or adding assertions that use these.